README.adoc

```diff
@@ -440,13 +440,18 @@ You will see the following output:
 Running it.io.openliberty.guides.system.SystemEndpointIT
 Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.4 sec - in it.io.openliberty.guides.system.SystemEndpointIT
 Running it.io.openliberty.guides.inventory.InventoryEndpointIT
+[WARNING ] Interceptor for {http://client.inventory.guides.openliberty.io/}SystemClient has thrown exception, unwinding now
+Could not send Message.
+[err] The specified host is unknown: java.net.UnknownHostException: UnknownHostException invoking http://badhostname:9080/inventory/properties: badhostname
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.264 sec - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 
 Results :
 
 Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning and error messages are expected and result from a request to a bad or an unknown host name. This request is made in the test `testUnknownHost()` from `InventoryEndpointIT`.
```

Comment:
Suggested update

>The warning and error messages are expected and result from a request to a bad or an unknown hostname. This request is made in the `testUnknownHost()` test from the `InventoryEndpointIT` endpoint.

- I think endpoint works after "`InventoryEndpointIT`". Please update to the correct noun if I got it wrong. Thx!

Reply:
@Charlotte-Holt 
I updated to 
> The warning and error messages are expected and result from a request to a bad or an unknown hostname. This request is made in the `testUnknownHost()` test from the `InventoryEndpointIT` intergation test.
what do you think?

Reply:
@gkwan-ibm Looks good. Just a typo in "intergation" - update to "integration".

Reply:
updated